### PR TITLE
negar/italian_enable

### PR DESCRIPTION
**Pull request by negar-binary**
enable italian language in display languages

### CODE REVIEW

File: src/javascript/binary/pages/trade/price.js [hunk 1]
```diff
@@ -180,22 +179,6 @@ var Price = (function () {
             description.innerHTML = '<div>'+proposal['longcode']+'</div>';
         }
 
-        if (document.getElementById('websocket_form')) {
```
Comment (raunakkathuria) [inline src/javascript/binary/pages/trade/price.js]: why these changes?

Comment (negar-binary) [inline src/javascript/binary/pages/trade/price.js]: @raunakkathuria JY asked that we don't hide the boxes anymore on invalid input for the new trading page.

Comment (raunakkathuria) [inline src/javascript/binary/pages/trade/price.js]: @negar-binary so we just show the invalid input error?

Comment (negar-binary) [inline src/javascript/binary/pages/trade/price.js]: @raunakkathuria from my understanding, yes
